pkg/interface/link/src/js/components/root.js

```diff
@@ -26,62 +33,167 @@ export class Root extends Component {
     let contacts = !!state.contacts ? state.contacts : {};
     const groups = !!state.groups ? state.groups : {};
 
+    const resources = !!state.resources ? state.resources : {};
     let links = !!state.links ? state.links : {};
     let comments = !!state.comments ? state.comments : {};
     const seen = !!state.seen ? state.seen : {};
 
+    const invites = '/link' in state.invites ?
+      state.invites['/link'] : {};
+
     return (
-      <BrowserRouter>
+      <BrowserRouter><Switch>
         <Route exact path="/~link"
           render={ (props) => {
             return (
               <Skeleton
-                active="channels"
```

Comment:
This `active` switch stuff was for mobile. I think `rightPanelHide` might make it all redundant, probably, but there should be a catch in `skeleton.js` that isn't there now? Now the root page on mobile looks like this:

<img width="499" alt="image" src="https://user-images.githubusercontent.com/20846414/76010140-69447200-5ee0-11ea-8665-c207e49dcd85.png">